### DISCUSSION TITLE
91C4E197-C967-4D42-9068-6EACE7F56741

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#128] [General]
# 91C4E197-C967-4D42-9068-6EACE7F56741

8 Ursae Minoris, also known formally as Baekdu, is a K-type giant star about 532.67 light-years away in the constellation Ursa Minor. It is orbited by a superjovian planet, named Halla, about 0.45 AU away from its host star. Both objects received their formal names during the second NameExoWorlds campaign in 2019

https://celestia.mobi/addon?item=91C4E197-C967-4D42-9068-6EACE7F56741

**Comment by exocritic5457** [comment]
What a strange system. Former binary?
